packages/EasyCodingStandard/README.md

```diff
@@ -260,6 +260,13 @@ parameters:
     cache_directory: .ecs_cache # defaults to sys_get_temp_dir() . '/_easy_coding_standard/_changed_files_detector_tests'
 ```
 
+#### How can I change the cache namespace?
+
+```yaml
+parameters:
+    cache_namespace: my_project_namespace # defaults to Strings::webalize(getcwd())'
+```
```

Comment:
Could you add situations, why would anyone try to do this? So people know when it's useful

Reply:
Improved description.

Reply:
Thanks